crypto/provider_conf.c

```diff
@@ -35,14 +35,14 @@ static int provider_conf_params(OSSL_PROVIDER *prov,
     STACK_OF(CONF_VALUE) *sect;
     int ok = 1;
 
-    OSSL_TRACE2(PROVIDER_CONF, "PROVIDER conf: %s = %s\n", name, value);
-
     sect = NCONF_get_section(cnf, value);
     if (sect != NULL) {
         int i;
         char buffer[512];
         size_t buffer_len = 0;
 
+        OSSL_TRACE1(CONF, "PROVIDER conf: start section %s\n", value);
```

Comment:
Just a first quick note: since the trace output already has a "CONF:" prefix (at least in the default setup of the application), I would just add "CONF: ...", or even better, "configuration: ..." in lower case.

Reply:
Sorry. I meant "PROVIDER: ", or "provider: "

Reply:
Surely, you mean that I should simply remove " conf"

Reply:
yup!

Reply:
I've changed the format strings to something more sensible, I hope